chapter05/qli/ex2.pl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+
+chomp( my @inputs = <STDIN> );
+
+print "1234567890"x5, "\n" ;
```

Comment:
actually I was thinking to do something like  `((1..9)."0")x5`, but I couldn't figure out how to write it properly, so end up typing every number from 1 to 9. :smile: